src/v/cluster/CMakeLists.txt

```diff
@@ -108,6 +120,9 @@ v_cc_library(
     metadata_dissemination_handler.cc
     metadata_dissemination_service.cc
     types.cc
+    remote_topic_properties.cc
+    topic_configuration.cc
+    topic_properties.cc
```

Comment:
does this need ` topic_properties.cc`? i'd figure it'd get it from `v::cluster_topic_properties`?

Reply:
Ah nice catch. The next commit introduces v_cluster_topic_properties but I forgot to remove these